src/Defra.Trade.ReMoS.AssuranceService.UI.Hosting/Pages/Registration/Assurances/UpdatedTermsAndConditions.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@page
+@using Defra.Trade.ReMoS.AssuranceService.UI.Hosting.Constants;
+@model Defra.Trade.ReMoS.AssuranceService.UI.Hosting.Pages.Registration.Assurances.UpdatedTermsAndConditions
+@{
+    ViewData["Title"] = "Updated Terms and conditions for the NI Retail Movement Scheme";
+}
+
+@section Back{
+    <a asp-page="@Routes.Pages.Path.BusinessListPath" class="govuk-back-link">Back</a>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ErrorSummary" model="ModelState" />
+
+        <h1 class="govuk-heading-xl">
+            Updated terms and conditions
+        </h1>
+        <p class="govuk-body">The <a href="~/terms-conditions" target="_blank" rel="noopener noreferrer">NI Retail Movement Scheme terms and conditions (opens in a new tab)</a> were updated on @Model.UpdatedTermsAndConditionsDate</p>
+        <p class="govuk-body">Share the scheme’s terms and conditions of the scheme with your business’s authorised representative.</p>
+
+        <form class="form" method="post">
+
+            <div class="govuk-form-group @(ModelState.HasError("TandCs") ? "govuk-form-group--error" : null)">
+                <p id="TandCs_Error" class="govuk-error-message" gds-validation-for="TandCs"></p>
+                <div class="govuk-checkboxes__item govuk-!-margin-bottom-6">
+                    <input class="govuk-checkboxes__input" id="TandCs" asp-for="TandCs" type="checkbox" value="true">
+                    <label class="govuk-label govuk-checkboxes__label govuk-!-font-weight-bold" for="TandCs">
+                        I confirm that @Model.PracticeName's authorised representative - @Model.AuthorisedSignatoryName - has read and understood the updated terms and conditions of the NI Retail Movement Scheme. @Model.AuthorisedSignatoryName is responsible for ensuring the requirements of the scheme are followed by the business.
+                        <p class="govuk-body"></p>
+                        <p class="govuk-body">Date of declaration: @DateTime.Now.ToString("d MMMM yyyy")</p>
```

Comment:
We use utc time so may be better to switch it to that

Reply:
changed